components/bill/invoice.templ

```diff
@@ -200,3 +203,12 @@ func isSimplifiedInvoice(doc doc.Document) bool {
 	}
 	return false
 }
+
+func isNonStandardInvoice(doc doc.Document) bool {
+	switch doc.GetType() {
+	case bill.InvoiceTypeCreditNote, bill.InvoiceTypeDebitNote, bill.InvoiceTypeCorrective, bill.InvoiceTypeProforma:
+	  return true
+	default:
+	  return false
+	}
+}
```

Comment:
`isNonStandardInvoice` works now that we don't need to negate it. I'd just abbreviate this function like this:

```suggestion
func isNonStandardInvoice(doc doc.Document) bool {
  return doc.GetType().In(bill.InvoiceTypeCreditNote, bill.InvoiceTypeDebitNote, bill.InvoiceTypeCorrective, bill.InvoiceTypeProforma)
}
```